config/sem_dedup_config.yaml

```
@@ -3,22 +3,23 @@ cache_dir: "semdedup_cache"
 num_files: 16
 
 # Embeddings configuration
-embeddings_save_loc: "embeddings"
 embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
 embedding_batch_size: 128
+embeddings_save_loc: "embeddings"
+embedding_pooling_strategy: "mean_pooling"
+embedding_column: "embeddings"
 write_embeddings_to_disk: true
+write_to_filename: false
 
 # Clustering configuration
-clustering_save_loc: "clustering_results"
-n_clusters: 1000
-seed: 1234
 max_iter: 100
-kmeans_with_cos_dist: false
-
-# Semdedup configuration
-which_to_keep: "hard"
-largest_cluster_size_to_process: 100000
+n_clusters: 1000
+clustering_save_loc: "clustering_results"
 sim_metric: "cosine"
+which_to_keep: "hard"
+sort_clusters: true
+kmeans_with_cos_dist: false
+partition_size: "2gb"
```

If we are adding this, lets be more explicit with this. 

```suggestion
clustering_input_partition_size: "2gb"
```